src/components/ComboButton/ComboButtonItem/ComboButtonItem.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * @file Combo button item.
+ * @copyright IBM Security 2019
+ */
+
+import React from 'react';
+import Button from '../../Button';
+import OverflowMenuItem from '../../OverflowMenuItem';
+import { namespace } from '../ComboButton';
+
+const ComboButtonItem = props => {
+  const { ...rest } = props;
+  return <span {...rest} className={`${namespace}-item`} />;
+};
+
+ComboButtonItem.defaultProps = {
+  ...Button.defaultProps,
```

Comment:
These props might be overwhelmingly long and conflicting possibly. It opens up way too much for a consumer to make sense of. For example `isDelete` does not apply to Buttons, but the prop types would say that it is a valid prop. It would force consumers to worry too much about the implementation of the `ComboButton`. I would just list out the specific ones that you need to render and nothing else.

They could just be: 
```js
{
  children: PropTypes.string.isRequired,
  renderIcon: PropTypes.node,
  onClick: PropTypes.onClick,
  disabled: PropTypes.bool,
  className: PropTypes.string,
}
```

Any other prop would just be confusing and unnecessary.